web/locales/en/plugin__monitoring-plugin.json

```diff
@@ -102,14 +115,14 @@
   "panel.styles attribute not found": "panel.styles attribute not found",
   "No data found": "No data found",
   "query results table": "query results table",
-  "Last {{count}} minute_one": "Last {{count}} minute",
-  "Last {{count}} minute_other": "Last {{count}} minutes",
-  "Last {{count}} hour_one": "Last {{count}} hour",
-  "Last {{count}} hour_other": "Last {{count}} hours",
-  "Last {{count}} day_one": "Last {{count}} day",
-  "Last {{count}} day_other": "Last {{count}} days",
-  "Last {{count}} week_one": "Last {{count}} week",
-  "Last {{count}} week_other": "Last {{count}} weeks",
+  "Last {{count}} minute_one": "Last {{count}} minute_one",
+  "Last {{count}} minute_other": "Last {{count}} minute_other",
+  "Last {{count}} hour_one": "Last {{count}} hour_one",
+  "Last {{count}} hour_other": "Last {{count}} hour_other",
+  "Last {{count}} day_one": "Last {{count}} day_one",
+  "Last {{count}} day_other": "Last {{count}} day_other",
+  "Last {{count}} week_one": "Last {{count}} week_one",
+  "Last {{count}} week_other": "Last {{count}} week_other",
```

Comment:
We need to keep the correct translations here

Reply:
I updated this in the latest commit.